colors.cjs

```diff
@@ -21,7 +21,7 @@ const colors = {
   },
   indigo: {
     50: '#eef2ff',
-    100: '#e0e7ff',
+    100: '#E0EAFF',
```

Comment:
I am not going to block on this, but the inconsistency between capital and lower case makes me sad.

Reply:
Fair, I just copied the values from Figma, and figured we will be revisiting this file once Design finalizes our color palette.